### DISCUSSION TITLE
what happens if stabilityDays is set and the depended project releases more often than stabilityDays?

### DISCUSSION THREAD

**Discussion by svenXY** [renovatebot/renovate#15991] [Request Help] [answered]
what happens if stabilityDays is set and the depended project releases more often than stabilityDays?

e.g. stabilityDays is set to 3 and the project releases daily.

I can imagine two possible outcomes:

1. since there are "older" versions that pass stabilityDays, renovate could use them instead and the PR would always be behind the releases by 3 days - this is what I would prefer
2. renovate always "finds" the latest version and will therefore never create PRs since the stabilityDays will never be reached.

I'm not clear as to what will happen, maybe someone with more insight can clarify.

Moreover - if 2 is true - wouldn't it make sense to implement something that makes it possible to install 3 day old releases of a fast-releasing project, even if newer ones already show up?

Thanks in advance,
Sven

**Comment by rarkins** [comment, marked as answer]
Check out internalChecksFilter=strict

**Reply by svenXY** [reply]
that's it! Cool, thanks!
